agent/pkg/models/models.go

```diff
@@ -17,13 +15,13 @@ func GetEntry(r *tapApi.MizuEntry, v tapApi.DataUnmarshaler) error {
 	return v.UnmarshalData(r)
 }
 
-func NewApplicableRules(status bool, latency int64, number int) tapApi.ApplicableRules {
-	ar := tapApi.ApplicableRules{}
-	ar.Status = status
-	ar.Latency = latency
-	ar.NumberOfRules = number
-	return ar
-}
+//func NewApplicableRules(status bool, latency int64, number int) tapApi.ApplicableRules {
```

Comment:
until the rules feature is implemented I kept it in comment (anyway it is not used right now) because the calling code in comment too (or not exists)